lib/main_development.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:bloc/bloc.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:mobile_test/app/app.dart';
+import 'package:mobile_test/app/app_bloc_observer.dart';
+import 'package:mobile_test/app/dependencies.dart';
+
+void main() {
+  Bloc.observer = AppBlocObserver();
+  setupDependencies();
+  FlutterError.onError = (details) {
+    log(details.exceptionAsString(), stackTrace: details.stack);
+  };
+
+  runZonedGuarded(
+    () => runApp(const SuperformulaApp()),
+    (error, stackTrace) => log(error.toString(), stackTrace: stackTrace),
+  );
```

Comment:
chore: The flavors are great but currently providing identical experiences. They should be removed or differentiated to justify their use.